pymilvus/client/grpc_handler.py

```diff
@@ -922,6 +922,23 @@ def _check():
 
         return Status(status.code, status.reason)
 
+    @retry_on_rpc_failure()
+    def alter_index(
+        self,
+        collection_name: str,
+        index_name: str,
+        extra_params: dict,
+        timeout: Optional[float] = None,
+        **kwargs,
+    ):
+        check_pass_param(collection_name=collection_name)
+        request = Prepare.alter_index_request(collection_name, index_name, extra_params)
```

Comment:
please check index_name a valid string and extra_params not None

Reply:
any criterion to check `index_name`?